scripts/postinstall.js

```diff
@@ -35,20 +35,53 @@ const packageName = 'timezone';
 // init cwd is the filepath of the initiating command
 const dirPath = `${process.env.INIT_CWD}/node_modules/${packageName}/`;
 
+let targetBlockValues = {};
 try {
   // eslint-disable-next-line global-require,import/no-dynamic-require
-  themesLocaleList = require(`${process.env.INIT_CWD}/src/blocks.json`).localeList;
+  targetBlockValues = require(`${process.env.INIT_CWD}/src/blocks.json`).values;
+
+  const defaultLocalizationObject = targetBlockValues.default.siteProperties.dateLocalization;
+
+  // must have a default language and timezone
+  // redefining locale list over the defaults
+  themesLocaleList = [defaultLocalizationObject.language];
+  targetTimeZones = [defaultLocalizationObject.timeZone];
+  /*
+    "sites": {
+      "arc-demo-1": {
+      "siteProperties": {
+        "dateLocalization": {
+          "language": "es",
+          "timeZone": "Europe/Madrid"
+        },
+      }
+    }
+  */
+
+  // if sites
+  // obj with key is truthy if sites property exists
+  Object.values(targetBlockValues.sites).forEach(({ siteProperties: sitePropertyObject }) => {
+    // if site has no date localization obj
+    if (sitePropertyObject.dateLocalization) {
+      themesLocaleList.push(sitePropertyObject.dateLocalization.language);
+      targetTimeZones.push(sitePropertyObject.dateLocalization.timeZone);
+    }
+  });
+
+  // dedupe languages and timezones if any added
+  themesLocaleList = [...new Set(themesLocaleList)];
+  targetTimeZones = [...new Set(targetTimeZones)];
 } catch (err) {
   // eslint-disable-next-line no-console
-  console.log(`${process.env.INIT_CWD}/src/blocks.json`, 'not found');
+  // console.log(`${process.env.INIT_CWD}/src/blocks.json`, 'not found');
```

Comment:
Is there a reason to hide this error and shouldn't it actually be that the file is not found, does not contain the proper defaults?  Can the defaults be excluded?

Given that, if the file is not found, we then want to fallback to the harcoded values instead of failing out?

Reply:
yeah I can keep the logging if it would be helpful. in practice, when we're installing this, nested dependency postinstall logging don't bubble up sadly

Reply:
Updated the fallback logic to define the defaults in the fallback rather than when the variable is defined 

![Uploading Screen Shot 2021-06-07 at 11.46.12.png…]()

Reply:
@nschubach also added a clarification for why no logging. if we want to keep it, I can add it back. I was mostly using it initially for debugging purposes but I wasn't sure what you felt

Reply:
@JackHowa No need, Was mainly curious if it would be helpful for debugging something that is wrong...

I don't foresee this being a problem since not many people will be building up a blocks.json from the ground up I assume.